src/vm/amd64/cgencpu.h

```diff
@@ -225,6 +215,18 @@ void UpdateRegDisplayFromCalleeSavedRegisters(REGDISPLAY * pRD, CalleeSavedRegis
 
 #define NUM_ARGUMENT_REGISTERS 4
 
+#define ENUM_CALLEE_SAVED_REGISTERS() \
+    CALLEE_SAVED_REGISTER(Rdi) \
+    CALLEE_SAVED_REGISTER(Rsi) \
+    CALLEE_SAVED_REGISTER(Rbx) \
```

Comment:
This order is hardcoded in assembly code at number of places, not just the places I am changing. I have added comment about it.

The order on Windows is what it used to be for a long time - not worth it to rock the boat to "fix it".

We need RBP to be last on Unix to erect proper RBP frames.